app/models/matchup.rb

```diff
@@ -16,14 +17,15 @@
 #  updated_at       :datetime         not null
 #
 class Matchup < ApplicationRecord
-  validates :year, :round, :conference, :number, :favorite_tricode, :underdog_tricode,
+  validates :sport, :year, :round, :conference, :number, :favorite_tricode, :underdog_tricode,
             :favorite_wins, :underdog_wins, :starts_at, presence: true
+  validates :sport, inclusion: {in: %w[nba mlb]}
   validates :year, numericality: {equal_to: 2021}
   validates :round, numericality: {
     greater_than_or_equal_to: 1, less_than_or_equal_to: 4
   }
-  validates :conference, inclusion: {in: %w[east west], if: -> { round < 4 }},
-                         exclusion: {in: %w[east west], if: -> { round == 4 }}
+  validates :conference, inclusion: {in: %w[east west AL NL], if: -> { round < 4 }},
+                       exclusion: {in: %w[east west AL NL], if: -> { round == 4 }}
```

Comment:
WS is round 3 right? So these won't work any more. (Or should the wild card games be considered the first round?)

I think you could add a method like this
```ruby
def final_round?
  (nba? && round == 4) ||
    (mlb? && round == 3)
end
```

And then you can do this
```suggestion
  validates :conference, inclusion: {in: %w[east west AL NL], unless: -> { final_round? }},
                       exclusion: {in: %w[east west AL NL], if: -> { final_round? }}
```

Reply:
I was thinking the wildcard round could be the first round for MLB. It's just a 1-game series with only one matchup per conference.

Reply:
I implemented your change - now we have the flexibility to change the number of rounds by sport